tests/smoke-tests/src/test/java/org/apache/activemq/artemis/tests/smoke/quorum/ZookeeperPluggableQuorumPeerTest.java

```diff
@@ -48,15 +47,15 @@ public ZookeeperPluggableQuorumPeerTest() {
       // both roles as both wish to be primary but will revert to backup
       primary = new BrokerControl("primary-peer-a", JMX_PORT_PRIMARY, "zkReplicationPrimaryPeerA", PRIMARY_PORT_OFFSET);
       backup = new BrokerControl("primary-peer-b", JMX_PORT_BACKUP, "zkReplicationPrimaryPeerB", BACKUP_PORT_OFFSET);
-      brokers = new LinkedList(Arrays.asList(primary, backup));
+      brokers = Arrays.asList(primary, backup);
    }
 
    @Test
    @Override
    public void testBackupFailoverAndPrimaryFailback() throws Exception {
       // peers don't request fail back by default
       // just wait for setup to avoid partial stop of zk via fast tear down with async setup
-      Wait.waitFor(this::ensembleHasLeader);
+      Assert.assertTrue(awaitAsyncSetupCompleted(TimeUnit.SECONDS.toMillis(30L), TimeUnit.MILLISECONDS));
```

Comment:
Being curious how you were able to remove the methods in the other class without affecting anything else, I had a look, and it appears like this different check will in fact do nothing really, as the called method just returns true. So its asserting true on something that just returns true.

The fact awaitAsyncSetupCompleted simply returns true, is interesting...though the effect of this change is still to no wait for startup, which it did before.

Reply:
Should I remove this dummy check everywhere?

Reply:
just @Ignore'd